ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/worker.rs

```diff
@@ -343,4 +362,8 @@ async fn process_streaming_response(
             },
         }
     }
+
+    // It is expected that we get to this point, the upstream server disconnects
+    // clients after 5 minutes.
+    Ok(())
```

Comment:
Yea the upstream may get disconnected; but it's most likely be indexer node update.